component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/cloud/DefaultChoreoFaultMetricEventBuilder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.am.analytics.publisher.reporter.cloud;
+
+import org.wso2.am.analytics.publisher.reporter.MetricSchema;
+
+import java.util.HashMap;
+
+/**
+ * Builder class for fault events
+ */
+public class DefaultChoreoFaultMetricEventBuilder extends DefaultFaultMetricEventBuilder {
+
+    protected DefaultChoreoFaultMetricEventBuilder() {
+        requiredAttributes = DefaultInputValidator.getInstance().getEventProperties(MetricSchema.ERROR);
```

Comment:
Should not this be MetricSchema.CHOREO_ERROR

Reply:
fixed with https://github.com/wso2/apim-analytics-publisher/pull/55/commits/4064e7e15db9c570a0e642eb42318d59f3b62100